# No request
---
string[] ids
geometry_msgs/Pose[] pose
shape_msgs/SolidPrimitive[] shape
